configs/config.hs

```diff
@@ -42,13 +42,15 @@ myMailKeybindings =
     [ Keybinding (EvKey (KChar 'a') []) (setTags [RemoveTag "inbox", AddTag "archive"] `chain` continue)
     ]
 
-writeMailtoFile :: Mail -> IO ()
+-- Note: The empty string return value is currently used as an error condition,
+-- but for this case it is not used.
```

Comment:
File a ticket to refactor this to return `IO (Either e ())` where `e ~ String` or maybe something else...

Reply:
See #200